### PR TITLE
Adding eli5 video the home page

### PR DESCRIPTION
**Pull request by dmitryvinn-fb**
## Motivation

Here at Meta Open Source, we've been creating short-form videos to explain Meta OSS projects in the simplest terms possible. We started embedding these videos into project pages as you can see in Hermes home page https://hermesengine.dev/

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

Yes
## Test Plan

The video plays and can be full-screened - manually tested
## Related PRs
None

### CODE REVIEW

**Comment by slorber** [comment]
Hi @dmitryvinn-fb 

We plan to redesign the homepage in https://github.com/facebook/docusaurus/issues/6386

In the meantime, I don't mind adding this ELI5 video to the homepage.

I'd prefer it to appear below user testimonials though, as providing reassurance looks more important to me than adding this video (but I may be wrong, quite subjective)

@yangshun any opinion?

**Comment by Josh-Cena** [comment]
Supabase puts it right in the banner where we have the jack-in-the-box Slash, so I would agree with putting it above the features as well

**Comment by dmitryvinn-fb** [comment]
Would you do it like this @yangshun? I wouldn't want to replace Docusaurus logo at the very top so this position might be a better one 
image

**Review by yangshun — APPROVED** [review]

**Comment by Josh-Cena** [comment]
@yangshun The merged version still has the video below the features🤦‍♂️ Not sure if you want that strongly.

**Comment by yangshun** [comment]
Oh oops. I'll tweak it later
